kiwi/tasks/base.py

```diff
@@ -280,7 +280,7 @@ def _ntuple_token(
 
         :rtype: list
         """
-        tokens = option.split(',', tuple_count - 1)
+        tokens = option.split(',', tuple_count - 1) if option else []
```

Comment:
This seems a fix for the commandline options in general not specifically for 'adding credentials to commandline'  feature ?

Reply:
correct, stumbled over it when testing and thought it can't hurt to fix it in the same slot